### DISCUSSION TITLE
self signed certificate issue.

### DISCUSSION THREAD

**Discussion by kungfusheep** [zed-industries/zed#6600] [Feature Requests]
Hi - My organisation uses a self-signed certificate internally which is causing issues trying to connect to zed's collaboration features.  The certificates are all registered correctly in my mac keychain. 

Does anyone have a work-around for this?

From the zed log:
`IO error: invalid certificate: UnknownIssuer: IO error: invalid certificate: UnknownIssuer: invalid certificate: UnknownIssuer`

**Comment by mark-pictor-csec** [comment]
sounds related to https://github.com/zed-industries/community/issues/1513

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
